package.json

```
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/Woorank/wappalyzer-wrapper",
   "license": "ISC",
   "dependencies": {
-    "wappalyzer": "5.6.3"
+    "wappalyzer": "^5.8.2"
```

No version ranges, this will break our package as soon as wappalyzer updates
```suggestion
    "wappalyzer": "5.8.2"
```